nvm.sh

```diff
@@ -2485,54 +2485,75 @@ nvm() {
       local LTS
       local NVM_UPGRADE_NPM
       NVM_UPGRADE_NPM=0
+      local PROVIDED_REINSTALL_PACKAGES_FROM
+      local REINSTALL_PACKAGES_FROM
+      local SKIP_DEFAULT_PACKAGES
+      local DEFAULT_PACKAGES
+
+      local provided_version=''
+      ADDITIONAL_PARAMETERS=''
+
       while [ $# -ne 0 ]
       do
         case "$1" in
           -s)
-            shift # consume "-s"
             nobinary=1
           ;;
           -j)
-            shift # consume "-j"
-            nvm_get_make_jobs "$1"
-            shift # consume job count
+            shift # 2 part parameter, comsume first part now
+            nvm_get_make_jobs "$1" # $1 is now second parameter piece
```

Comment:
it looks like the job count is now consumed into ADDITIONAL_PARAMETERS?

Reply:
ADDITIONAL_PARAMETERS is only added to if the parameter does meet the other cases above it. The job count parameter is it's own case. So it is not added to ADDITIONAL_PARAMETERS.

Reply:
which line handles that? note that `nvm install` and `nvm use`, with no "provided version" argument (but still with possible additional parameters) are also valid use case (they'll read from `.nvmrc`)